README.md

```diff
@@ -48,25 +48,51 @@ pip to install directly from the master branch on github:
 pip install git+https://github.com/keras-team/keras-cv.git tensorflow --upgrade
 ```
 
-## Using KerasCV with Keras Core
+## Configuring your backend
 
-As of version `0.6.0`, KerasCV supports multiple backends with Keras Core out of 
-the box. There are two ways to configure KerasCV to run with multi-backend 
-support:
+**Keras 3** is an upcoming release of the Keras library which supports
+TensorFlow, Jax or Torch as backends. This is supported today in KerasNLP,
+but will not be enabled by default until the official release of Keras 3. If you
+`pip install keras-cv` and run a script or notebook without changes, you will
+be using TensorFlow and **Keras 2**.
 
-1. Via the `KERAS_BACKEND` environment variable. If set, then KerasCV will be 
-using Keras Core with the backend specified (e.g., `KERAS_BACKEND=jax`).
-2. Via the `.keras/keras.json` and `.keras/keras_cv.json` config files (which 
-are automatically created the first time you import KerasCV):
-   - Set your backend of choice in `.keras/keras.json`; e.g., `"backend": "jax"`. 
-   - Set `"multi_backend": True` in `.keras/keras_cv.json`.
+If you would like to enable a preview of the Keras 3 behavior, you can do
+so by setting the `KERAS_BACKEND` environment variable. For example:
 
-Once that configuration step is done, you can just import KerasCV and start 
+```shell
+export KERAS_BACKEND=jax
+```
+
+Or in Colab, with:
+
+```python
+import os
+os.environ["KERAS_BACKEND"] = "jax"
+
+import keras_cv
+```
+
+> [!IMPORTANT]
+> Make sure to set the `KERAS_BACKEND` before import any Keras libraries, it
+> will be used to set up Keras when it is first imported.
+Until the Keras 3 release, KerasCV will use a preview of Keras 3 on PyPI named
+[keras-core](https://pypi.org/project/keras-core/).
+
+> [!IMPORTANT]
+> If you want to import Keras and guarantee that you're getting the same version
+> that KerasCV is using, just use `from keras_cv.backend import keras`. This
```

Comment:
I think we should not use this, as it is not available in our exported API, or would be only via `from keras_cv.src.backend import keras`.